src/context.rs

```diff
@@ -368,4 +399,4 @@ impl PySessionContext {
         let df = PyDataFrame::new(wait_for_future(py, result).map_err(DataFusionError::from)?);
         Ok(df)
     }
-}
+}
```

Comment:
FYI: you have removed the line at the very end, which is probably something your editor did automatically. That might be the problem with the styling.

Reply:
Sorry I was traveling. Thanks for pointing out.

Reply:
Sorry for the late reply, I was traveling. Looks like I had an issue with `cargo fmt`. Thanks for the pointer.